src/config.ts

```diff
@@ -454,11 +457,18 @@ export const ASSET_SWAPPER_MARKET_ORDERS_V1_OPTS: Partial<SwapQuoteRequestOpts>
     sampleDistributionBase: 1.05,
     feeSchedule: FEE_SCHEDULE_V1,
     gasSchedule: GAS_SCHEDULE_V1,
+    exchangeProxyOverhead: (sourceFlags: number) =>
```

Comment:
Do we need to add something like this to the `V0` opts or will it just default to `0`? Is `0` correct in that case?

Reply:
0 is correct in that case